src/Surfnet/StepupGateway/SecondFactorOnlyBundle/Service/LoaAliasLookupService.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupGateway\SecondFactorOnlyBundle\Service;
+
+use Surfnet\StepupBundle\Value\Loa;
+use Surfnet\StepupGateway\GatewayBundle\Exception\InvalidArgumentException;
+
+final class LoaAliasLookupService
+{
+    /**
+     * @var array<string,string>
+     */
+    private $loaAliases;
+
+    /**
+     * @param array<string,string> $loaAuthnContextClassMapping
+     */
+    public function __construct(array $loaAliases)
```

Comment:
I think you renamed AuthnContextClass mappings to aliases at some point, but there seem to be some references to the old name (see docblock, InvalidArgumentException, SurfnetStepupGatewaySecondFactorOnlyExtension::replaceAuthnContextClassRefLookupMapping)